docs/01-readme.md

```diff
@@ -141,6 +141,20 @@ If Awesome does not start or the configuration file is not producing the
 desired results the user should examine this file to gain insight into the
 problem.
 
+# Awesome not an option in your desktop selection? Try this.
+In order for your login screen or, to be more precise, your display manager, to display this button, it needs to find an awesome.desktop file inside /usr/share/xsessions/. When installing AwesomeWM from source, this file is not installed automatically.
+
+Run the following command to create the file:
+
+`echo '[Desktop Entry]
```

Comment:
this is not correct markdown syntax for multiline codeblocks

Reply:
(you current version will eat up the linebreaks after rendering)

Reply:
fixed, thank you for catching this mistake.


"
### Awesome not an option in your desktop selection? Try this.

In order for your login screen or, to be more precise, your display manager, to display this button, it needs to find an awesome.desktop file inside /usr/share/xsessions/. When installing AwesomeWM from source, this file is not installed automatically.

Run the following command to create the file:

```
echo '[Desktop Entry]
Name=awesome
Comment=Highly configurable framework window manager
TryExec=awesome
Exec=awesome
Type=Application
' | sudo tee /usr/share/xsessions/awesome.desktop
```
"